.github/ISSUE_TEMPLATE/BUGS.yml

```diff
@@ -0,0 +1,51 @@
+name: "Bug Report 🐛"
```

Comment:
To be honest I don't think we need a "bug report" template for docs. A "content report" issue is what's relevant for docs. So maybe turning this to a link to report any technical issues upstream (i.e. `tauri-apps/tauri`). If there are "technical" issues or bugs then we still have the "other" template for those to filter into.

<img width="1039" alt="Screenshot 2023-09-07 at 08 46 41" src="https://github.com/tauri-apps/tauri-docs/assets/15347255/ecf34d36-a8a2-48c8-a885-9451dfae5465">